src/modules/core/index.ts

```diff
@@ -96,7 +98,16 @@ function init(config: ServiceWorkerConfiguration = {}) {
   });
 }
 
+function forcedNullifcation() {
+  import(/* webpackChunkName: "service-worker-remover" */ '../service-worker-remover/index').then(
+    async ({ ServiceWorkerRemover }) => {
```

Comment:
```typescript
async ({ ServiceWorkerRemover }) => {
  new ServiceWorkerRemover().installNoOpServiceWorker();
},
```

Can be rewritten as...
```typescript
async ({ ServiceWorkerRemover }) => new ServiceWorkerRemover().installNoOpServiceWorker(),
```

Reply:
are you aware of a prettier rule for this?
This definitely looks like an optimization that should either be done by prettier or the compiler.

Reply:
manually fixing this for now